routes/definitions.js

```diff
@@ -70,6 +70,28 @@ async function listDefinitions(request, response) {
   // if running on localhost, allow a force arg for testing without webhooks to invalidate the caches
   const force = request.hostname.includes('localhost') ? request.query.force || false : false
   const expand = request.query.expand === '-files' ? '-files' : null // only support '-files' for now
+  try {
```

Comment:
Are these changes really related to this problem? I'm not a fan of this code being in the route (perhaps better in the service itself) but either way, it seems unrelated to the issue at hand

Reply:
It is. The problem was we weren't capturing the the errors and returning them in a way that was useful to the user. I added this with the purpose of capturing the errors when they happen during a call and then returning them to the user. Otherwise it would just error out with a 500 whenever an error happened somewhere in the callstack.

Reply:
I am happy to jump on a call and talk through this more if it would be useful.

Reply:
This looks similar to the code in the next method, maybe we want to use a common function to do this consistently instead of duplicating?

Reply:
What happened was I forgot to delete the original code when I moved it into the try catch block. It's fixed!